sycl/test-e2e/InvokeSimd/Feature/invoke_simd_struct_by_pointer.cpp

```diff
@@ -0,0 +1,169 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+//
+// VISALTO enable run
+// RUN: env IGC_VISALTO=63 IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+
+// Tests invoke_simd support in the compiler/headers
+
+/* Test case specification:
+ * -----------------------
+ * Test passing pointers to structs to invoke_simd functions.
+ *
+ * Test case description:
+ * -----------------------------
+ * This is a simple test case that defines a struct named
+ * multipliers with 2 int members, x and y. Suppose these
+ * members represent values by which we want to scale the
+ * addtion of vectors A + B. That is, we want to compute
+ * C = A + B * x * y (the sum of A + B scaled by x,
+ * and then scaled by y).
```

Comment:
> the sum of A + something

reads weird to me. Change s/+/and/ maybe?

Reply:
Changed the text, hope it is clearer now.